src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,45 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final Pattern CUSTOM_PATTERN = Pattern.compile("//(.)\n(.*)");
+    private static final int DELIMITER_GROUP = 1;
+    private static final int TEXT_GROUP = 2;
+
+    private StringAddCalculator() {}
+
+    public static int splitAndSum(String text) {
+        if (hasNullOrEmpty(text)) { return 0; }
+        String[] numbers = splitTextToNumbers(text);
+        return sum(numbers);
+    }
+
+    private static boolean hasNullOrEmpty(String text) {
+        return text == null || text.equals("");
+    }
```

Comment:
```suggestion
    private static boolean hasNullOrEmpty(String text) {
        return text == null || text.isBlank();
    }
```

equals로 빈 문자열과 같은지 비교하는 방식보다 제공되는 메서드(isEmpty, isBlank)를 사용하면 좋을 것 같습니다.

Reply:
피드백 감사합니다!